next/components/paragraph.tsx

```diff
@@ -4,6 +4,8 @@ import { ParagraphText } from "@/components/paragraph--text";
 import { ParagraphVideo } from "@/components/paragraph--video";
 import { Paragraph } from "@/lib/zod/paragraph";
 
+import { ParagraphAccordion } from "./paragraph--accordion";
```

Comment:
⚠️ This "paragraph mother" component will eventually have a lot of imported paragraph components. Some of them might import heavy dependencies (example markdown parsers) and add a lot of possible unnecessary weight to the js bundle.

💡 Consider importing these using Next.js [Dynamic Imports](https://nextjs.org/docs/advanced-features/dynamic-import) to only import the component if it's actually used on the page. Maybe not **all** because it causes loading waterfalls, but at least the heavy ones.

I'll add an interesting Twitter thread to the bargain!
It's related to server components, but Dan uses this same kind of use case as an example. Totally worth a read! _(I had to read it few times to digest it)_ https://twitter.com/dan_abramov/status/1629597621126922241

Reply:
thanks juho! we will have a look at this

Reply:
I made a new story to add this, so we don't forget and we can still merge this one :) 

https://wunder.atlassian.net/browse/NEX-46